lang/en.yml

```diff
@@ -379,7 +379,7 @@ en:
     PASSWORD: Password
     PASSWORDEXPIRED: 'Your password has expired. Please choose a new one.'
     PLURALNAME: Members
-    REMEMBERME: 'Remember me next time?'
+    REMEMBERME: 'Remember me next time? (for %d days on this device)'
```

Comment:
"Keep me signed in" as been added and "Remember me next time (after x days)" has been assigned as the title attribute to the checkbox (so it shows when the mouse is over the box)